lib/SPIRV/libSPIRV/SPIRVOpCodeEnum.h

```diff
@@ -438,3 +438,4 @@ _SPIRV_OP(LoopControlINTEL, 5887)
 _SPIRV_OP(ReadPipeBlockingINTEL, 5946)
 _SPIRV_OP(WritePipeBlockingINTEL, 5947)
 _SPIRV_OP(FPGARegINTEL, 5949)
+_SPIRV_OP(ExpectINTEL, 9997)
```

Comment:
What does 9997 means?

Reply:
I guess, it means 5631 :)

Reply:
Oh, that an artifact from a "prior values reservation" age. Thanks!